core/domain/story_fetchers_test.py

```diff
@@ -250,20 +252,30 @@ def test_get_multi_users_progress_in_stories(self) -> None:
             user_stories_progress[0]['completed_node_titles'], ['Title 1'])
         self.assertEqual(user_stories_progress[0]['topic_name'], 'Topic')
 
-    def test_get_story_summary_by_id(self) -> None:
+    # Here we use object because we want to mock the 'get' attribute.
+    @mock.patch.object(
+        story_models.StorySummaryModel, 'get'
+    )
+    def test_get_story_summary_by_id(self, mock_get: mock.Mock) -> None:
+        mock_story_summary = mock.Mock()
+        mock_story_summary.id = self.story_id
+        mock_story_summary.title = 'Title'
+        mock_story_summary.description = 'Description'
+        mock_story_summary.node_titles = ['Title 1']
+        mock_story_summary.thumbnail_bg_color = None
+        mock_story_summary.thumbnail_filename = None
+
+        mock_get.side_effect = [mock_story_summary, None]
```

Comment:
This seems a bit odd to me. Isn't the mock summary supposed to be the return value, not a side effect?